Documentation/quickstart.md

```diff
@@ -1,19 +1,174 @@
 ---
 title: Quickstart
-weight: 200
+weight: 300
 ---
 
-# Quickstart Guides
+{% include_relative branch.liquid %}
 
-Welcome to Rook! We hope you have a great experience installing the Rook **cloud-native storage orchestrator** platform to enable highly available, durable storage
-in your Kubernetes cluster.
+# Ceph Quickstart
+
+Welcome to Rook! We hope you have a great experience installing the Rook **cloud-native storage orchestrator** platform to enable highly available, durable Ceph storage in your Kubernetes cluster.
 
 If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://rook-io.slack.com). You can sign up for our Slack [here](https://slack.rook.io).
 
-Rook provides a growing number of storage providers to a Kubernetes cluster, each with its own operator to deploy and manage the resources for the storage provider.
+This guide will walk you through the basic setup of a Ceph cluster and enable you to consume block, object, and file storage
+from other pods running in your cluster.
+
+## Minimum Version
+
+Kubernetes **v1.11** or higher is supported by Rook.
+
+**Important** If you are using K8s 1.15 or older, you will need to create a different version of the Rook CRDs. Create the `crds.yaml` found in the [pre-k8s-1.16](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/pre-k8s-1.16) subfolder of the example manifests.
+
+## Prerequisites
+
+To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [follow these instructions](pre-reqs.md).
+
+In order to configure the Ceph storage cluster, at least one of these local storage options are required:
+- Raw devices (no partitions or formatted filesystems)
+  - This requires `lvm2` to be installed on the host.
+    To avoid this dependency, you can create a single full-disk partition on the disk (see below)
+- Raw partitions (no formatted filesystem)
+- Persistent Volumes available from a storage class in `block` mode
+
+## TL;DR
+
+If you're feeling lucky, a simple Rook cluster can be created with the following kubectl commands and [example manifests](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph).
```

Comment:
The "feeling lucky" lacks a bit of confidence even though it probably has a humoristic tone. Should we keep it?

Reply:
Yeah I think it's better to remove